specification/v3.5/OSDM-online-api-v3.5.0.yml

```diff
@@ -1961,6 +1961,7 @@ paths:
     post:
       tags:
         - Bookings Search
+        - Search
```

Comment:
This is the first time probably we have multiple tags used and also tags not listed in the beginning of the openapi. Could have impact on code generation if the logic doesn't consistently take the first, and globally listed tag.

Reply:
i have added the tag descriptions. Tags are used for documentation "Common use cases for tags include grouping endpoints together in navigation sections of API reference documentation, and providing a way to include or exclude a subset of endpoints from an OpenAPI description.". This should not affect code generation except in case you use it to generate parts of the api (e.g. generate all masterdata endpoints).